backend/src/controllers/clients.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { Response } from 'express';
+import { AuthRequest } from 'common/auth.middleware';
+import { IClient } from "shared-types";
+import { BaseController } from "./base.controller";
+import clientModel from "../models/client.model";
+import httpStatus from 'http-status';
+import { isEmpty } from 'lodash';
+import { signTokens } from '../utils/tokens';
+
+class ClientsController extends BaseController<IClient> {
+  constructor() {
+    super(clientModel);
+  }
+
+  async updateByBusinessId(req: AuthRequest, res: Response){
+    const fields = ['businessName', 'businessDescription'];
+    const { businessId } = req.body;
+    this.debug(`Updating by bid - ${businessId}`);
+
+    if(fields.some((field) => isEmpty(req.body[field].toString()))){
```

Comment:
You can use zod schemas (like in shared types schema.ts file) to validate the data automatically with a middle that @chenyair implemented

Reply:
You mean to create a schema for this param.body and it will autocheck if its empty with the middleware?

Reply:
It moved to the `server-utils` package